cassandra-sidecar-release/prepare_release.sh

```diff
@@ -0,0 +1,433 @@
+#!/bin/bash
+
+#set -o xtrace
+
+##### BEFORE YOU BEGIN #####
+
+# 1. Ensure your gpg configuration in ~/.gradle/gradle.properties
+#    For example:
+#    signing.gnupg.keyName=<your-key>
+#    signing.gnupg.passphrase=<your-passphrase>
+#    #signing.gnupg.executable=gpg # optional
+#    #signing.gnupg.useLegacyGpg=true # optional
+#    #signing.gnupg.homeDir=gnupg-home # optional
+#    #signing.gnupg.optionsFile=gnupg-home/gpg.conf # optional
+
+# 2. Ensure your maven credentials are configured in ~/.gradle/gradle.properties
+#    For example:
+#    maven.repository.url=https://repository.apache.org/service/local/staging/deploy/maven2
+#    maven.username=<asf-username>
+#    maven.password=<asf-password>
```

Comment:
these types of general improvements, can we get them in the server release scripts too ? 🙏